composer.json

```diff
@@ -8,23 +8,28 @@
         {
             "name": "Kauan Sousa",
             "email": "kauanslr.ks@gmail.com",
-            "role": "Original Maintainer"
+            "role": "Maintainer"
+        },
+        {
+            "name": "Leonardo Lemos",
+            "email": "leonardo.lemos@convenia.com.br",
+            "role": "Maintainer"
         }
     ],
     "require": {
         "php": ">=7.1.3",
         "ext-json": "*",
         "ext-sockets": "*",
         "php-amqplib/php-amqplib": ">=2.9",
-        "laravel/framework": "^8.0",
+        "laravel/framework": "^5.6|^6.0|^7.0|^8.0",
```

Comment:
Do we still need `^5.6` support as there's no more community support?

Reply:
We dont need, but supporting 5.6 does not add maintenance effort, we just need to keep 5.6 entry on `composer.json`. @kauanslr what do you think?